Sources/TuistSupport/Logging/Logger.swift

```diff
@@ -53,6 +57,9 @@ public enum LogOutput {
             handler = StandardLogHandler.self
         case .json:
             handler = JSONLogHandler.self
+        case .quiet:
+            LoggingSystem.bootstrap(quietLogHandler(label:))
+            return
```

Comment:
I'm open to more elegant implementation suggestions 🙏

Reply:
I think this is fine, let's keep the early return 👍